docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 db:
-  image: postgres:9.4.1
+  image: postgres:9.4.4
   ports:
-    - "5432:5432"
+    - "5433:5432"
```

Comment:
Good move. This bothered me but I didn't take the opportunity to fix it at the time. Thank you for doing this!